pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/project.json

```diff
@@ -2,36 +2,36 @@
   "dependencies": {
     "Microsoft.Net.Native.Compiler": "1.5.0",
     "Microsoft.NETCore.Portable.Compatibility": "1.0.2",
-    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.3",
```

Comment:
Lets please not update Microsoft.NETCore.UniversalWindowsPlatform or Microsoft.NETCore at this point. We are trying to be intentional about which packages we actually update in these projects.